config/vendor-api-v1.yml

```diff
@@ -305,8 +305,10 @@ paths:
     post:
       tags:
       - Testing
-      summary: Regenerate test data
+      summary: Regenerate test data (DEPRECATED)
       description: |
+        This endpoint has been deprecated and will return an error. Please use `/test-data/clear` and `/test-data/generate`.
+
         Deletes all applications and generates 100 new applications. Only
         available on the Sandbox.
```

Comment:
Shall we remove this bit of the description since the endpoint doesn't do this anymore?